app/controllers/staff/reports_submissions_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Staff::ReportsSubmissionsController < Staff::BaseController
+  include Secured
+
+  def show
+    report = Report.find(params[:report_id])
+    authorize report
+    @report_presenter = ReportPresenter.new(report)
+    render "staff/reports/submit"
+  end
+
+  def update
+    @report = Report.find(params[:report_id])
+    authorize @report
+
+    @report.state = :submitted
+    if @report.valid?
+      @report.save!
+      @report.create_activity key: "report.submitted", owner: current_user
+      redirect_to complete_report_submit_path(@report)
+    else
```

Comment:
How would a user end up here, given all the policy checking etc beforehand?

Reply:
It does seem unlikely this branch could be triggered as the only state change that has been made is something we know to be valid. If we accept `state` as a parameter though, this will become necessary us the user may submit an unrecognised state, or one that is not a valid transition from the report's current state.